tests/bluetooth/tester/src/gap.c

```diff
@@ -1094,6 +1137,12 @@ uint8_t tester_init_gap(void)
 {
 	int err;
 
+	cb.pairing_accept = auth_pairing_accept;
+
+	if (bt_conn_auth_cb_register(&cb)) {
```

Comment:
`bt_conn_auth_cb_register` is a kind of unique way to register callbacks for bluetooth, as only one structured is allowed to be registered. 

Instead of failing the test case when the return code is non-zero, perhaps we could validate that it is non-zero and only if the code isn't `-EALREADY` we fail the test?